lib/game_runner.js

```diff
@@ -75,8 +76,9 @@ GameRunner.prototype.playGame = function (ship, lifeCount, projectiles){
 
 GameRunner.prototype.newLevel = function (ship, lifeCount, projectiles) {
   var level = this.level += 1;
+  var asteroidCount = 8 + 2*level;
   $(".level").html("Level "+ level)
-  asteroidHelper.addAsteroids(8, asteroids);
+  asteroidHelper.addAsteroids(asteroidCount, asteroids);
```

Comment:
Level 1 still starts with 5; this comes into play starting at level 2? Fine either way, just confirming my understanding.